packages/react/src/modules/slack/components/ConnectedSlackChannelPicker/ConnectedSlackChannelPicker.tsx

```diff
@@ -0,0 +1,216 @@
+import Knock, { User } from "@knocklabs/client";
+import { useMemo, useState } from "react";
+import { ContainerObject, useSlackChannels, Channel } from "../..";
+
+import "./styles.css";
+
+type Props = {
+  user: User;
+  tenant: string;
+  connectionsObject: ContainerObject;
+  host?: string;
+  userToken: string;
+  knockSlackChannelId: string;
+  knockClientId: string;
+};
+
+export const ConnectedSlackChannelPicker: React.FC<Props> = ({
+  user,
+  tenant,
+  connectionsObject,
+  knockClientId,
+  host = "api.knock.app",
+  userToken,
+  knockSlackChannelId,
+}) => {
+  const knockClient = useMemo(() => {
```

Comment:
Could we use the knock client from `useKnock` (from the `<KnockProvider />`). I think it's safe to assume that provider will be wrapping any usage of the slack components and could simplify these components